Formula/gping.rb

```diff
@@ -12,7 +12,7 @@ class Gping < Formula
   # https://github.com/Homebrew/homebrew-core/pull/66366#discussion_r537339032
   livecheck do
     url :stable
-    regex(/^v?(\d+(?:\.\d+)+)$/i)
+    regex(/^gping-v?(\d+(?:\.\d+)+)$/i)
```

Comment:
```suggestion
    regex(/^gping[._-]v?(\d+(?:\.\d+)+)$/i)
```

We use `[._-]` as the generic delimiter between the software name and numeric version in `livecheck` block regexes, so it's good to follow suit.

Reply:
Fixed! Thank you